### ISSUE TITLE
Blog site accessibility bugs

### ISSUE THREAD

**Issue by chris-brookes** [redbadger/website-honestly#585]
**Description of the issue:**
The blog pages have multiple fails on the Tenon.io accessibility test.

1. Multiple items are missing an `alt` attribute:
- Article's main image.
- Arrow image by the "More from xxxxx" link.
- Twitter and Facebook images in the 'Share this article' section. (this section is shown twice on the tablet/desktop page).
- Map pin next to the the address in the footer.
 
2. The image has `alt` and `title` attributes that are different:
- Header logo.
- Footer images: Github, Twitter, Slack, LinkedIn, Instagram, Facebook, YouTube and Google+.
 
3. Embedded frames (including YouTube videos) do not have a `title` attribute.
 

See a Tenon.io review at https://goo.gl/hh6cLm. Change viewport size in options and re-analyse the page address to test all 3 layouts.
(You can ignore the error results for TestID 57 in this review. This is a bug on Tenon's side)

**Comment by dantrain** [comment]
Fixed by https://github.com/redbadger/blog-squarespace-template/pull/89.
Can't stop people making "Click here!" links though 😄
